release notes/v0.37.0.md

```diff
@@ -0,0 +1,73 @@
+k6 v0.37.0 is here! 🎉 Mainly it contains fixes and ongoing efforts with refactoring.
+
+## New Features!
+
+### Added an option to output k6 logs to a file through --log-output ([#2285](https://github.com/grafana/k6/pull/2285))
+
+This is on top of the supported stdout/stderr and loki instance.
+
+The feature speaks for itself with examples of usage:
+
+```sh
+k6 run --log-output file=./k6.log --logformat json ./examples/stages.js
+```
+
+with a defined minimal log level:
+
+```sh
+k6 run --log-output file=./k6.log,level=info --logformat json ./examples/stages.js
+```
+
+Thanks, @[alyakimenko](https://github.com/alyakimenko) for the contribution!
+
+**Docs**: [Using file output](https://k6.io/docs/using-k6/options/#file)
+
+### Added experimental basic event loops ([#882](https://github.com/grafana/k6/issues/882))
+
+We added basic event loop functionality ([#2228](https://github.com/grafana/k6/pull/2228) and [#2373](https://github.com/grafana/k6/pull/2373)). This is a first step and currently isn't used by anything, but is now available to extensions. In following releases this will be used by k6 and will mean that some APIs will start having callbacks or return Promises. But first it's likely that it will be used in some experimental extensions.
+
+If you are an extension developer please use it and give your feedback. But take into consideration that it's likely that the API may change
+
+## Extensions
+
+`v0.37.0` finalizes ([#2376](https://github.com/grafana/k6/pull/2376) and [#2384](https://github.com/grafana/k6/pull/2384)) the switching of our internal modules to [a new Go/JavaScript module API](https://k6.io/docs/extensions/guides/create-an-extension/#advanced-javascript-extension). As the last internal the GRPC module was refactored ([#2365](https://github.com/grafana/k6/pull/2365)).
+
+It's important to highlight that the old API is deprecated and will be removed in the next k6's release (`v0.38.0`). Every extension that isn't using that API will get a warning message, like:
+
+```sh
+WARN[0000] Module 'k6/x/sql' is using deprecated APIs that will be removed in k6 v0.38.0, for more details on how to update it see https://k6.io/docs/extensions/guides/create-an-extension/#advanced-javascript-extension
+```
+
+We did migrations for some xk6 extensions (see connected issues to the task [#2344](https://github.com/grafana/k6/issues/2344)). The pull requests can serve as examples on how to transition your extension to the new API.
+
+## Docker Repository
+
+We migrated our docker hub's repository from `loadimpact/k6` to the [grafana/k6](https://hub.docker.com/r/grafana/k6) ([#2377](https://github.com/grafana/k6/pull/2377)).
+
+```sh
+docker run -i grafana/k6 run - <script.js
+```
+
+We continue pushing the images to the `loadimpact/k6` for some time, but if you use it in your local or CI environments please plan the migration.
```

Comment:
```suggestion
We will continue to push `loadimpact/k6` images for some time, but if you use it in your local or CI environments please plan the migration.
```

It would be good to mention here the time frame we plan to continue to do so. Several versions, a few months...?

Reply:
We don't have a date defined, but it's better to attract attention because as far as I know the `loadimpact/k6` probably have some pull limitation :thinking:

Reply:
I think any specifier would be better than an indeterminate "some time". Maybe "for several more releases"?